pkg/vm/engine/tae/txn/txnimpl/node.go

```diff
@@ -516,7 +516,10 @@ func (n *insertNode) Window(start, end uint32) (bat *gbat.Batch, err error) {
 		if err != nil {
 			return nil, err
 		}
-		srcVec, _ := src.Window(start, end+1).CopyToVector()
+		srcVec, err2 := src.Window(start, end+1).CopyToVector()
+		if err2 != nil {
+			panic(err)
```

Comment:
You sure you want to panic err here?   Not err2?

Reply:
Ok. I'll panic err2.